rxbinding-design/build.gradle

```diff
@@ -42,6 +42,7 @@ android {
   lintOptions {
     textReport true
     textOutput 'stdout'
+    disable 'ResourceType', 'PrivateResource'
```

Comment:
That is not the information I'm after. I want to know the cause in the library which triggers the lint errors. Either the lint errors are wrong and should be suppressed with a link to a bug reporting the failure or the lint errors are accurate and the errant code corrected.